dns/dnscrypt-proxy/Makefile

```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		dnscrypt-proxy
-PLUGIN_VERSION=		1.7
+PLUGIN_VERSION=		1.8
 PLUGIN_REVISION=	1
```

Comment:
if we bump version we remove revision

```suggestion
```

Reply:
Do I remove the whole line or just leave it line as PLUGIN_REVISION=